sormas-app/app/src/main/res/layout/fragment_event_read_layout.xml

```diff
@@ -209,6 +210,28 @@
                 style="@style/ControlSingleColumnStyle"
                 app:value="@{data.srcType}" />
 
+            <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal">
+
+
+                <de.symeda.sormas.app.component.controls.ControlTextReadField
+                        android:id="@+id/event_srcInstitutionalPartnerType"
+                        style="@style/ControlFirstOfTwoColumnsStyle"
+                        app:value="@={data.srcInstitutionalPartnerType}"
```

Comment:
Read-only fields can't have an "=" sign after the @ sign (because this is the trigger for two-way databinding)